### ISSUE TITLE
require 'stringio'

### ISSUE THREAD

**Issue by shokai** [hardbap/firmata#8]
Hi

when call Firmata::Board#connect, it raise error

```
uninitialized constant Firmata::Board::StringIO (NameError)
```

<img src="http://gyazo.com/f9303ad35acfaa6b6d7ecf5452edc132.png">
## require 'stringio'

this worked.

``` ruby
require 'rubygems'
require 'firmata'
require 'stringio'

arduino = Firmata::Board.new '/dev/tty.usbmodem621'
arduino.connect
```

**Comment by hardbap** [comment]
Thanks for the patch! There seems to be a difference between Ruby 1.9.2 (which you are using) and 1.9.3 (which I'm using) because there is no error with 1.9.3. I'll add a note that this lib targets 1.9.3.

**Comment by hardbap** [comment]
Fixed in #9462d33.
